Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1499,8 +1494,7 @@ public void Logout(string accessToken)
             {
                 Limit = 1,
                 AccessToken = accessToken
-
-            }).Items.FirstOrDefault();
+            }).Items[0];
```

Comment:
This will throw an exception when the query returns no elements

Reply:
good point